CONTRIBUTING.md

```diff
@@ -2,8 +2,9 @@ TigerOS
 ====================
 
 ## Contributing
-Please fork this repository and submit pull requests against the devel branch. Pull requests will be reviewed as time permits.
-Depending on the change you may also fork the repo and submit your PR against a new feature branch that you created. This is preferred if adding new code or items and not just modifying existing contents of the repo.
+Please fork this repository and submit pull requests against the devel branch. When you do work in your fork, create a new branch for that work rther than wroking in a branch that has the same name as on in our repository. This helps to reduce occurences of the two repositories getting out of sync. For instructions on configuring your git environment to match with our workflow please see [here](https://help.github.com/articles/fork-a-repo/#keep-your-fork-synced) Pull requests will be reviewed as time permits.
```

Comment:
Rather and working are misspelled. "...in the upstream branch" would be clearer than "in a branch that has the same name as on in our respository [sic]." Should make clear that the new branch is local and should not be pushed upstream to us. The two sentences on being in sync cleaned up, they're very wordy and I feel like they're redundant. Something along the lines of "This helps prevent issues when merging your PRs." and putting an "additional reading" list below which could have that article and the other one.